### PR TITLE
fix build error caused by cp swagger dir

### PR DESCRIPTION
**Pull request by urfreespace**
fixed build error, detail https://github.com/apache/pulsar-site/actions/runs/3317429325/jobs/5480293467

### CODE REVIEW

**Comment by urfreespace** [comment]
@tisonkun PTAL, thanks

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
It seems the result https://github.com/apache/pulsar-site/commit/f17fd0698c2400bb69cef8aeba20c5f49b289e15 is under `content/content/build` which has two more level inner then the expected one.

File: site2/tools/publish-website.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ PULSAR_SITE_TMP=/tmp/pulsar-site
       | grep -v $PULSAR_SITE_TMP/content/.htaccess \
       | xargs rm -rf
   fi
-  cp -r $GENERATED_SITE_DIR/content/ $PULSAR_SITE_TMP/content
+  cp -R $GENERATED_SITE_DIR/content/ $PULSAR_SITE_TMP/content
```
Comment (tisonkun) [inline site2/tools/publish-website.sh:68]: The root cause should be that we change from `-r` to `-R` and the behavior changed.

File: site2/tools/build-site.sh [hunk 2]
```diff
@@ -50,7 +50,7 @@ bash scripts/split-version-build.sh $@
 cd "$WEBSITE_DIR"
 
 mkdir -p "$GEN_SITE_DIR"/reference
-cp -r docsify/* "$GEN_SITE_DIR"/reference
+cp -R docsify/* "$GEN_SITE_DIR"/reference
```
Comment (tisonkun) [inline site2/tools/build-site.sh:53]: And thus we should also take care of this line.

**Pull request merged**